app/code/Magento/AdminAnalytics/Test/Mftf/Test/AdminCheckAnalyticsTrackingTest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCheckAnalyticsTrackingTest">
+        <annotations>
+            <stories value="AdminAnalytics Check Tracking."/>
+            <title value="AdminAnalytics Check Tracking."/>
+            <description value="AdminAnalytics Check Tracking."/>
+            <severity value="MINOR"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+            <magentoCLI command="config:set admin/usage/enabled 1" stepKey="enableAdminUsageTracking"/>
+            <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
+                <argument name="tags" value="config full_page"/>
+            </actionGroup>
+            <reloadPage stepKey="pageReload"/>
+        </before>
+        <after>
+            <magentoCLI command="config:set admin/usage/enabled 0" stepKey="disableAdminUsageTracking"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <waitForPageLoad stepKey="waitForPageReloaded"/>
```

Comment:
I usually recommend putting such actions to a separate action group. However, in this particular case, I don't think that the newly introduced action group will be reusable somehow. So, we may leave it as it is